src/Makefile

```diff
@@ -21,6 +21,15 @@ CHECK_CENTOS5 := $(shell sh -c 'grep "CentOS release 5." /etc/redhat-release 2>&
 
 ARCH_FLAGS =
 
+DBG_SYM_SUPPORT?=yes
+ifneq (${uname_S},Linux)
+ifneq (${uname_S},Darwin)
+ifneq (${TARGET},winagent)
```

Comment:
I am not sure, but can we use the & operand and remove 2 `ifneq` statements? This change improves readability

Reply:
The alternative is use nested filter-out function, but more difficult to read. There's no logic operators available to use